src/test/resources/services-local.conf

```diff
@@ -14,23 +14,22 @@
 
 # This file is read when runLocal = true in application.conf (default)
 
-services {
-    auth-login-stub {
-      protocol = http
-      host = localhost
-      port = 9949
-    }
-    address-lookup-frontend {
-      protocol = http
-      host = localhost
-      port = 9028
-      postcode = "ZZ9Z 9TT"
```

Comment:
Variables for specific requests should be passed into the function(s) in the simulation file rather than defined here.